src/utils/mode/impl/process/agent.ts

```
@@ -0,0 +1,119 @@
+
+    if (process.platform === 'win32') {
+      forkOptions.windowsHide = true;
+    }
+
+    // add debug execArgv
+    const debugPort = process.env.EGG_AGENT_DEBUG_PORT ?? 5800;
+    if (this.options.isDebug) {
+      forkOptions.execArgv = process.execArgv.concat([ `--inspect-port=${debugPort}` ]);
+    }
+
+    const agentProcess = this.#agentProcess = fork(this.getAgentWorkerFile(), args, forkOptions);
```

Biome (1.9.4)

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)